src/core/Button/Button.js

```diff
@@ -3,27 +3,17 @@ import PropTypes from 'prop-types'
 import Icon from '../Icon'
 import s from './styles'
 
-function Button({
-    role,
-    type,
-    size,
-    kind,
-    icon,
-    label,
-    active,
-    disabled,
-    onClick,
-}) {
+function Button({ role, type, kind, icon, label, active, disabled, onClick }) {
```

Comment:
should make the *role* as `role="button"` inside the Button or do you see any use case for Button assuming a different **role**

Reply:
Agreed, we can just default to `role="button"`.